.gitignore

```diff
@@ -9,6 +9,10 @@ package-lock.json
 test/images/
 tmp/
 
+# Cofiguration directory of IDE and editor
```

Comment:
typo: Configuration

Reply:
:) She didn't mean that you should literally put there `typo:` she was just saying you mistyped `Configuration`. So can you please put there `# Configuration directory of IDE and editor`

Reply:
:joy: OK, re-push